dynamic-layers/virtualization/recipes-azure-iot/iotedge/aziot-edged/aziot-edged-1.5.0.inc

```diff
@@ -1,7 +1,7 @@
-# override rolling SRCREV  "relase/1.4" of iot-identity-service dependency
+# override rolling SRCREV  "1.5" of iot-identity-service dependency
```

Comment:
right now they using the main branch to assign the TAG 1.5

Reply:
irrelevant??? 
how does iotedge reference iot-identity-service crates? by tag? this would be preferred and then it would **not** be a rolling srcrev anymore

Reply:
change comment to "main".
iotedge reference now with: e.g. 
_aziot-identity-common_ = { git = "https://github.com/Azure/iot-identity-service", branch = "main" }